test-requirements.txt

```diff
@@ -5,3 +5,8 @@ mock
 nose
 pep8
 sphinx
+pytest
```

Comment:
Is there a reason you are switching everything to pytest instead of sticking with unittest?